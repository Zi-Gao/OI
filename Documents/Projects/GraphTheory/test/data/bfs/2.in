112 188
20 91 11427
29 91 2225
24 81 18536
98 70 12014
110 55 17089
78 38 8548
106 33 31847
33 97 17181
13 43 22356
21 14 30931
12 53 2909
67 67 843
40 11 11905
104 11 24780
11 19 27262
19 47 1526
101 78 1601
20 41 30878
41 13 17732
3 103 14212
69 59 30732
29 110 28662
49 29 16437
66 107 24256
5 12 20428
90 75 27644
16 111 7924
41 61 10646
5 21 1024
56 81 27309
87 58 6538
72 57 15886
82 37 15151
104 27 16507
46 42 27893
20 37 2950
73 6 2703
10 23 3106
110 76 29425
42 9 22580
49 27 23030
52 6 15564
80 81 19718
2 82 18138
15 5 6400
61 88 1640
14 20 28370
14 30 20682
9 86 12673
90 50 24647
20 65 5015
11 99 28701
91 103 25389
20 89 27127
1 6 27399
89 9 30254
67 79 29829
7 24 21092
104 70 2382
14 5 19037
57 104 25860
98 26 7468
85 101 26208
48 35 3329
92 88 3210
96 17 25153
83 84 30852
37 44 23214
45 62 21511
80 41 25541
93 88 13857
88 24 11176
57 100 29548
69 65 5665
83 106 2759
63 67 2982
86 16 24782
96 84 30311
95 84 2944
59 45 9674
98 41 23772
55 13 19177
41 18 30296
75 90 24800
86 56 18562
45 22 10777
55 83 18730
32 88 2007
29 19 12254
96 38 25507
90 107 31660
4 14 14972
103 55 13922
10 14 10185
105 82 10429
17 12 8829
47 46 20554
38 92 2301
67 70 13037
10 93 10396
97 92 30017
36 66 13252
62 8 12365
65 17 9778
68 104 14414
15 41 30921
93 91 10616
102 105 20199
79 29 28931
79 94 1344
76 62 30690
56 18 20143
95 47 25065
17 111 25981
108 48 20319
89 85 22535
12 109 2085
53 15 11732
92 20 16118
44 8 7793
6 86 111
64 76 26034
52 99 13542
97 88 31021
77 72 11572
19 29 4348
84 31 20985
89 77 10808
73 26 2569
56 73 29607
79 89 6152
22 88 1417
75 12 22611
73 45 27544
34 22 16334
67 81 29834
82 15 4404
18 87 31933
40 27 16881
1 25 4565
58 99 18616
87 51 27500
80 29 24398
38 21 22832
77 97 28224
81 51 251
45 1 13443
15 15 10185
88 36 5677
42 9 22175
3 96 9410
56 88 2229
105 48 14544
103 83 17033
15 15 10196
99 64 2470
79 110 4754
88 43 17214
41 3 18283
92 63 5100
106 12 4447
37 62 1334
38 77 22444
49 12 1454
78 7 31575
37 70 24248
43 96 14076
105 33 1897
90 58 15031
9 80 24053
111 82 8458
40 2 22952
54 19 31492
106 56 17762
110 109 24860
90 30 5892
23 33 889
21 85 4698
66 80 28574
110 4 9822
75 57 31411
36 25 27123
6 11 18151
1 108 22876
3 28 32666
56 46 2079
61 47 31566
79 80 29473
