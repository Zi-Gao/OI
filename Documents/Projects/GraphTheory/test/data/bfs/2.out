1 3 4 5 6 7 8 9 11 12 13 14 15 16 17 18 19 20 21 22 24 25 26 27 28 29 30 31 33 35 36 37 38 41 42 43 44 45 46 47 48 51 53 55 56 57 58 61 62 63 64 65 66 67 70 72 73 76 77 78 79 80 81 82 83 84 85 86 87 88 89 91 92 94 96 97 99 100 101 103 104 106 107 108 109 110 111 